opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -1,622 +1,179 @@
-# WEB-INF/configuration.properties instead.
-opennms.rtc-client.http-post.base-url = http://localhost:8980/opennms/rtc/post
+### RTC IPC
 
-# The username the RTC uses when authenticating itself in an HTTP POST.
+opennms.rtc-client.http-post.base-url = http://localhost:8980/opennms/rtc/post
 opennms.rtc-client.http-post.username = rtc
-
-# The password the RTC uses when authenticating itself in an HTTP POST.
 opennms.rtc-client.http-post.password = rtc
 
-###### MAP IPC ######
-# The base of a URL that Map System clients use when creating a Map subscription URL.
-# If you are using Tomcat instead of the built-in Jetty, change this in
-# WEB-INF/configuration.properties instead.
-opennms.map-client.http-post.url = http://localhost:8980/opennms/map/post
+### Map IPC
 
-# The username the Map System uses when authenticating itself in an HTTP POST.
+opennms.map-client.http-post.url = http://localhost:8980/opennms/map/post
 opennms.map-client.http-post.username = map
-
-# The password the Map System uses when authenticating itself in an HTTP POST.
 opennms.map-client.http-post.password = map
 
-###### JETTY WEB UI ######
-# If you are using Jetty, this is the port to listen on
+### Jetty
+
 org.opennms.netmgt.jetty.port = 8980
-# If you want Jetty with AJP support, this is the port to listen on
 #org.opennms.netmgt.jetty.ajp-port = 8981
-# By default, Jetty will listen on all interfaces. You can set a specific
-# bind address here. If you set this to a value other than 127.0.0.1,
-# you will need to update the rtc-client and map-client URLs above.
 #org.opennms.netmgt.jetty.host = 127.0.0.1
-# 
-# This enables NCSA style request logging in ${opennm.home}/logs
-#
-# Uncomment this and change to true to enable this
 #org.opennms.netmgt.jetty.enableRequestLogging=false
-
-# This sets the maximum size for a form submission in jetty.  
-# The default value is 200000 bytes.  Setting it to -1 disables
-# the form limit
-# In 1.8 or earlier set
 #org.mortbay.jetty.Request.maxFormContentSize=200000
-#
-# in 1.9 or later set 
 #org.eclipse.jetty.server.Request.maxFormContentSize=200000
-
-# This sets the request header size for jetty.
-# The default value is 4000 bytes.
 #org.opennms.netmgt.jetty.requestHeaderSize=4000
-
-# This sets the maximum number of items that can be in web forms (like the
-# Provisioning web UI)
 org.eclipse.jetty.server.Request.maxFormKeys=2000
-
-###### JETTY HTTPS SUPPORT ######
-# Details: http://www.opennms.org/index.php/Standalone_HTTPS_with_Jetty
-# If you want Jetty to provide an HTTPS listener, this is the port to listen on
-# Note that setting this property does NOT disable the plain HTTP listener,
-# which is required by Rtcd to post realtime status updates.  If you do not
-# wish to allow unsecured HTTP access to the OpenNMS web UI, you must set
-# org.opennms.netmgt.jetty.host above or use firewall rules to accomplish this.
 #org.opennms.netmgt.jetty.https-port = 8443
```

Comment:
The Jetty HTTPS stuff needs to be in its own section apart from normal Jetty properties. Also, some of this might be obsolete since we now use jetty.xml to run the server. Only the properties that are referenced in the default jetty.xml file should be necessary now.